book/src/user/requirements.md

```diff
@@ -1,16 +1,22 @@
 # System Requirements
 
-We recommend the following requirements for compiling and running `zebrad`:
+Zebra has the following hardware requirements.
+
+## Recommended Requirements
 
 - 4 CPU cores
 - 16 GB RAM
-- 300 GB available disk space for building binaries and storing cached chain
-  state
+- 300 GB available disk space 
 - 100 Mbps network connection, with 300 GB of uploads and downloads per month
 
-Zebra's tests can take over an hour, depending on your machine. Note that you
-might be able to build and run Zebra on slower systems — we haven't tested its
-exact limits yet.
+## Minimal Requirements
+
+- 2 CPU cores
+- 4 GB RAM
```

Comment:
I think in theory, Zebra may currently crash with any less than 20GB of RAM in circumstances where it's tracking 100 chains with similar tip heights, though that's very unlikely to happen on Mainnet.

We may want to consider allowing users to configure the maximum number of chain forks that Zebra tracks to avoid that remote possibility.